events.html

```diff
@@ -16,3 +16,55 @@
         </div>
     </div>
 </div>
+<div class="container list-events">
+    <section class="row col-12">
+        <div class="row-header col-12">
+            <h2>Events</h2>
+            <ul class="nav nav-tabs">
+                <li class="nav-item">
+                    <a class="nav-link active">Upcoming Events</a>
+                </li>
+                <li class="nav-item"> 
+                    <a class="nav-link disabled">Past Events</a>
+                </li>
+            </ul>
+        </div>
+        <div class="row-content col-12">
+            <div class="card-deck">
+                {% assign count = 0 %}
+                {% for events in site.data.events %}
```

Comment:
For example that is what you need: 

``` 
{% for event in site.data.events %}
      {% include event.html %}
``` 

Create the html in `_includes/event.html` with this: 

``` 
                        <div class="card-column col-12 col-lg-6">
                            <article class="card-announcement event card card-m {{event.category}}">
                                <div class="card-header">
                                    <div class="col-12 col-sm-9 col-lg-8">
                                        <div class="card-category">{{ event.category_name }}</div>
                                        <div class="card-date"><div class="material-symbols-outlined" aria-hidden="true">
                                            <span class="material-symbols-outlined">calendar_month</span></div>{{ event.date }}, {{ event.time }}</div>
                                        <h3 class="card-title">{{ events.title }}</h3>
                                        {% if event.host %}
                                        <p class="card-host">Host: {{ event.host }}</p>
                                        {% endif %}
                                        {% if event.speaker %}
                                        <p class="card-speaker">Speaker: {{ event.speaker }}</p>
                                        {% endif %}
                                    </div>
                                    <div class="card-media col-12 col-sm-3 col-lg-4">
                                        <img src="{{ event.image.path }}" alt="{{ event.image.alt }}">
                                    </div>
                                </div>
                                <div class="card-body col-12">
                                    <p>{{ event.description }}</p> 
                                </div>
                                <div class="card-footer col-12">
                                    <a href="{{ event.link.path }}" target="{{ event.link.target }}" class="btn btn-outline-primary">{{event.link.title }}</a>
                                </div>
                            </article>
                        </div>
``` 

Basically you can reuse in home events, and this page, also for past and future events ;)